ethexe/service/src/tests.rs

```diff
@@ -72,7 +72,7 @@ async fn basics() {
         sequencer: Default::default(),
         validator: Default::default(),
         validator_session: Default::default(),
-        max_commitment_depth: 1_000,
+        max_eth_sync_depth: 1_000,
```

Comment:
not in this place, but tbh for prod I'd put much bigger amount. something like 10k blocks from eth

Reply:
or even maybe 50k

Reply:
it's already 100k for production see DEFAULT_MAX_DEPTH , you set it))

Reply:
)))